lib/node_modules/@stdlib/fs/append-file/docs/types/test.ts

```diff
@@ -0,0 +1,219 @@
+	appendFile.sync( true, 'beepboop' ); // $ExpectError
+	appendFile.sync( null, 'beepboop' ); // $ExpectError
+	appendFile.sync( undefined, 'beepboop' ); // $ExpectError
+	appendFile.sync( [], 'beepboop' ); // $ExpectError
+	appendFile.sync( {}, 'beepboop' ); // $ExpectError
+	appendFile.sync( ( x: number ): number => x, 'beepboop' ); // $ExpectError
+}
+
+// The compiler throws an error if the `sync` method is provided a second argument which is not a string or buffer...
+{
+	appendFile.sync( '/path/to/beepboop', false ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', true ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', null ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', undefined ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', [] ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', {} ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', ( x: number ): number => x ); // $ExpectError
+}
+
+// The compiler throws an error if the `sync` method is provided an options argument which is not an object or string...
+{
+	appendFile.sync( '/path/to/beepboop', 'beepboop', null ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', true ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', false ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', 123 ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', [] ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', ( x: number ): number => x ); // $ExpectError
+}
+
+// The compiler throws an error if the `sync` method is provided an `encoding` option which is not a string or null...
+{
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { encoding: 123 } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { encoding: true } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { encoding: false } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { encoding: [] } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { encoding: {} } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', {
+		encoding: ( x: number ): number => x,
+	} ); // $ExpectError
+}
+
+// The compiler throws an error if the `sync` method is provided a `flag` option which is not a string...
+{
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: 123 } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: true } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: false } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: null } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: [] } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', { flag: {} } ); // $ExpectError
+	appendFile.sync( '/path/to/beepboop', 'beepboop', {
+		flag: (x: number): number => x,
```

Comment:
Same thing. We don't split over lines like this in test.ts files. As the saying goes, "when in Rome..."